diff -urN base-passwd-3.5.29.org/group.master base-passwd-3.5.29/group.master
--- base-passwd-3.5.29.org/group.master	2017-07-03 14:40:19.393127378 +0800
+++ base-passwd-3.5.29/group.master	2017-07-03 15:05:17.827057104 +0800
@@ -40,5 +40,6 @@
 users:*:100:
 sdcard:*:1015:diag
 diag:*:53:
+qti:*:54:
 rebooters:*:1301:diag
 nogroup:*:65534:
diff -urN base-passwd-3.5.29.org/passwd.master base-passwd-3.5.29/passwd.master
--- base-passwd-3.5.29.org/passwd.master	2017-07-03 14:40:19.389127359 +0800
+++ base-passwd-3.5.29/passwd.master	2017-07-03 14:42:30.073745023 +0800
@@ -16,4 +16,5 @@
 irc:*:39:39:ircd:/var/run/ircd:/bin/sh
 gnats:*:41:41:Gnats Bug-Reporting System (admin):/var/lib/gnats:/bin/sh
 diag:*:53:53:diag:/nonexistent:/bin/sh
+qti:*:54:54:qti:/nonexistent:/bin/sh
 nobody:*:65534:65534:nobody:/nonexistent:/bin/sh
